python/src/MemoizeEvaluation_doc.i.in

```diff
@@ -0,0 +1,175 @@
+%feature("docstring") OT::MemoizeEvaluation
+"Evaluation which keeps tracks of input and output.
+
+Usually we use :class:`~openturns.MemoizeFunction` that relies
+on a :class:`~openturns.Function` passed as argument, and store
+input and output :class:`~openturns.Sample`.
+It also has a caching behavior, enabled by default.
```

Comment:
So how to choose between `MemoizeEvaluation` and `MemoizeFunction`? What is the difference?

Reply:
The same as `Function` and `Evaluation`